sdk/src/client/node_api/core/routes.rs

```diff
@@ -398,6 +413,18 @@ impl ClientInner {
             .await
     }
 
+    /// Get all UTXO changes of a given slot by its index.
+    /// GET /api/core/v3/commitments/by-index/{index}/utxo-changes
+    pub async fn get_utxo_changes_by_slot_index(&self, slot_index: &SlotIndex) -> Result<UtxoChangesResponse> {
```

Comment:
`SlotIndex` is `Copy`, so the APi is probably more convenient if it's not taking a `&`?

Reply:
Definitely, copy pasted mistake

Reply:
Changed